packages/select/src/ux-select.ts

```diff
@@ -355,13 +356,16 @@ export class UxSelect implements UxInputComponent {
     }
     this.isCollapsing = true;
     this.optionCtEl.classList.remove('ux-select__list-container--open');
+    const listTransitionString = getComputedStyle(this.element).getPropertyValue('--aurelia-ux--select-list-transition')
+      || UxSelectTheme.DEFAULT_LIST_TRANSITION;
+    const listTransition = parseInt(listTransitionString.replace('ms', ''));
```

Comment:
If we are using `parseInt`, probably no need to do a replace here, since it's at the end of the string. But maybe need to look at the config value normalization story a bit more across components for transition.